ergotree-interpreter/src/eval/savltree.rs

```diff
@@ -189,6 +265,161 @@ mod tests {
             unreachable!();
         }
     }
+    proptest! {
+        #[test]
+        fn eval_avl_properties(v in any::<AvlTreeData>(), new_ops in any::<AvlTreeFlags>(), new_digest in any::<ADDigest>()) {
```

Comment:
If this test failed it'd be hard to pinpoint the failing case, especially looking at the logs on CI. Consider splitting it into separate tests or providing an error message to identify which case is failed.

Reply:
I split it into separate tests.